concordia/admin/__init__.py

```diff
@@ -150,26 +151,25 @@ def get_urls(self):
 
 @admin.register(Resource)
 class ResourceAdmin(admin.ModelAdmin, CustomListDisplayFieldsMixin):
-    list_display = ("campaign", "sequence", "title", "resource_url")
+    list_display = ("campaign", "topic", "sequence", "title", "resource_url")
+    list_display_links = ("campaign", "topic", "sequence", "title")
+
+
+@admin.register(Topic)
+class TopicAdmin(admin.ModelAdmin):
```

Comment:
This should use `prepopulated_fields` to automatically set the slug from the title when creating a new topic

Reply:
I thought I already added that? If not then I will.